### ISSUE TITLE
Readme.md doesn't have ubuntu install commands unlike other toolchains (minor issue)

### ISSUE THREAD

**Issue by zecoxao** [ps2dev/ps2toolchain#100]
it should have a line such as

apt install gcc make patch git texinfo flex bison libgsl-dev libgmp-dev libmpfr-dev libmpc-dev gettext libnewlib-dev libipt-dev babeltrace

and/or, alternatively

apt install clang make patch git texinfo flex bison libgsl-dev libgmp-dev libmpfr-dev libmpc-dev gettext libnewlib-dev libipt-dev babeltrace

the last 3 are very likely required as the config option complains about them when building the toolchain

**Comment by fjtrujy** [comment]
You're right, however, this repo is not the one that you should install for preparing the PS2 Environment; the one you need to use is:
https://github.com/ps2dev/ps2dev
Over there, you have all the required Ubuntu libraries and commands to be executed.
